### PR TITLE
Jett tech

### PR DESCRIPTION
**Pull request by JettTech**

### CODE REVIEW

**Review by JettTech — COMMENTED** [review]
Please approve so I can complete the pull...

File: models/user.js [hunk 1]
```diff
@@ -1,18 +1,64 @@
...
+    primaryKey: true,
+    defaultValue: DataTypes.UUIDV4, //set to REFERENCE the Universal-Unique-ID est above... 
+    allowNull: false
+  },
+  name: {
+    type: DataTypes.STRING,
+    required: true
+  },
+  gender: {
+    type: DataTypes.ENUM,
```
Comment (ChadGatling) [inline models/user.js:14]: We are switching this to be just lowercase strings. male, female for now

File: models/user.js [hunk 2]
```diff
@@ -1,18 +1,64 @@
...
+  },
+  age: {
+    type: DataTypes.INTEGER,
+    allowNull: false
+  },
+  phoneNumber: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  genderPreference: {
```
Comment (ChadGatling) [inline models/user.js:26]: same as the gender. also be care full about changing variable names. I have already established these column names

File: models/user.js [hunk 3]
```diff
@@ -1,18 +1,64 @@
...
+  agePrefFloor: DataTypes.INTEGER,
 
-  Author.associate = function(models) {
-    // Associating User with Info(maybe if thats what we end up doing)
-    // When an User is deleted, also delete any associated Info
-    Author.hasMany(models.Data, {
-      onDelete: "cascade"
-    });
-  };
+  nativeLang: {
```
Comment (ChadGatling) [inline models/user.js:35]: lets just match the strings. no need for numbers

**Review by ChadGatling — CHANGES_REQUESTED** [review]
Need to delete or untrack your config file. pull the latest master and use the latest directory. keep things simple dont add extra work for people with extra folders and things. drop your html files. austin and lorna already worked to make them. thats their job.

**Pull request closed without merging**